app/viewModels/transportEquipment/index/ApplyAnotherItemViewModel.scala

```diff
@@ -67,28 +68,29 @@ object ApplyAnotherItemViewModel {
           case (_, i) =>
             val itemIndex = Index(i)
 
-            val changeOrRemoveUrl: String = mode match {
-              case CheckMode =>
-                controllers.transportEquipment.index.routes.GoodsReferenceController.onSubmit(ArrivalId(arrivalId), equipmentIndex, itemIndex, mode).url
-              case NormalMode => "" //TODO Some(routes.RemoveItemController.onPageLoad(departureId, mode, equipmentIndex, itemIndex).url)
-            }
+            def itemPrefix(item: String) = messages("transport.item.prefix", item)
 
-            val changePrefix: String = mode match {
-              case CheckMode  => "site.edit"
-              case NormalMode => "site.delete"
+            def buildListItem(item: Item): ListItem = mode match {
```

Comment:
Just as a note, happy with what you have - we could also look into doing something such as

`def buildListItem(item: Item, mode: Mode): ListItem = {
  val name = itemPrefix(item.toString)
  val (changeUrl, removeUrl) = mode match {
    case CheckMode =>
      (
        Some(controllers.transportEquipment.index.routes.GoodsReferenceController.onSubmit(ArrivalId(arrivalId), equipmentIndex, itemIndex, mode).url),
        None
      )
    case NormalMode =>
      (
        None,
        Some("") //TODO Some(routes.RemoveItemController.onPageLoad(departureId, mode, equipmentIndex, itemIndex).url)
      )
  }
  ListItem(name, changeUrl, removeUrl)
}
`

Reply:
Ah sorry, I was quick to merge after the approval as a couple tickets were being held up by my changes. I'll bear this ^ in mind though, cheers mate!